index.js

```diff
@@ -17,8 +17,14 @@ function LessCompiler (sourceTrees, inputFile, outputFile, options) {
   if (!(this instanceof LessCompiler)) {
     return new LessCompiler(sourceTrees, inputFile, outputFile, options)
   }
+  
+  var cacheOptions = merge({
+    filterFromCache: {
+      include: [/.*\.less$/]
```

Comment:
Yeah not sure, we probably should add `.css` as well, since you can import css. Will make the change. Do globs work for 1.1.0? Also with globs would it be `['*.less', '*.css']`?